# requested command
HybridImpedanceCommandTrajPosMsg command
---
# current command
HybridImpedanceCommandTrajPosMsg command